xwiki-platform-core/xwiki-platform-livetable/xwiki-platform-livetable-ui/src/main/resources/XWiki/LiveTableResultsMacros.xml

```diff
@@ -739,8 +739,11 @@
  * NOTE: This macro uses variables defined in livetable_filterProperty . It was not meant to be used alone.
  *#
 #macro (livetable_filterStringListProperty)
-  #set ($whereSql = "${whereSql} and concat('|', concat(upper(${safe_tableAlias}.textValue), '|')) like upper(?)")
-  #set ($discard = $whereParams.add("%|$filterValue|%"))
+  #set ($filterValues = $request.getParameterValues($colname))
+  #foreach ($filterValue in $filterValues)
+    #set ($whereSql = "${whereSql} and concat('|', concat(upper(${safe_tableAlias}.textValue), '|')) like upper(?)")
```

Comment:
It can't be configured visually with the default column filters. However, it could be configured:

1. Through a hidden input, based on a parameter set in the livetable column options. This means that it can't be toggled by the user in the UI, but it can still be configured by the livetable author when the data semantics makes it more logical to filter on ALL or ANY of the values.
2. Something that very few people are aware of, the livetable can have an advanced filters zone, where more advanced filtering is possible. See for example [this page](https://playground.phenotips.org/AllData), expand "Advanced data filters", and look at the filters for "By clinical diagnosis" where the switch is possible.